src/Aspire.Cli/Backchannel/BackchannelDataTypes.cs

```diff
@@ -1,6 +1,8 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+using System.Text.Json.Serialization;
```

Comment:
We should consider renaming this file to ExtensionBackchannelDataTypes to avoid confusion with the backchannel datatypes we share between CLI and Apphost.